api/routes/testAPI.js

```diff
@@ -37,7 +37,7 @@ const FormData = sequelize.define('FormData', {
   },
 });
 
-// Sync the model with the database (create the table if it doesn't exist)
+
```

Comment:
before syncing u have to connect to database... first of all connection to database is made then table is created... do this